docs/source/_templates/sidebar.html

```diff
@@ -0,0 +1,99 @@
+<nav data-toggle="wy-nav-shift" class="pytorch-left-menu" id="pytorch-left-menu">
+    <div class="pytorch-menu pytorch-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
+      <div class="sidebar">
+    <div class="pytorch-left-menu-search">
+        {% include "searchbox.html" %}
+    </div>
+
+    <!-- render sidebar from the toctree -->
+    {% set global_toc = toctree(maxdepth=1,
+                                collapse=theme_collapse_navigation|tobool,
+                                includehidden=theme_includehidden|tobool) %}
+    {% if global_toc %}
+        {{ global_toc }}
+    {% endif %}
+
+    <!-- PREVIOUS RELEASES -->
+        {% if version_list %}
+            <div class="sidebar-l1-expandable">Previous Releases</div>
+            <div class="sidebar-subheadings">
+                {% for version in version_list | sort(reverse=True) %}
+                    <div class="sidebar-l2"><a href="/ecosystem/retworkx/stable/{{ version }}/index.html">{{ version }}</a></div>
```

Comment:
The href is changed.

Reply:
This actually should be:

```suggestion
                    <div class="sidebar-l2"><a href="/ecosystem/rustworkx/stable/{{ version }}/index.html">{{ version }}</a></div>
```

Thanks to the project rename and then the ecosystem migration we've had 3 copies of the docs out there: `documentation/retworkx`, `documentation/rustworkx`, and now `ecosystem/rustworkx` the last one should be the canonical copy moving forward.